supabase/migrations/20260824134415_credit_auto_top_up.sql

```
@@ -0,0 +1,339 @@
+-- Auto top-up settings on orgs, plus apply remaining credits after a partial debit.
+
+ALTER TABLE "public"."orgs"
+  ADD COLUMN IF NOT EXISTS "auto_top_up_enabled" boolean DEFAULT false NOT NULL,
+  ADD COLUMN IF NOT EXISTS "auto_top_up_threshold" numeric(18,6) DEFAULT 10 NOT NULL,
+  ADD COLUMN IF NOT EXISTS "auto_top_up_last_attempt_at" timestamp with time zone;
+
+ALTER TABLE "public"."orgs"
+  DROP CONSTRAINT IF EXISTS "orgs_auto_top_up_threshold_min";
+
+ALTER TABLE "public"."orgs"
+  ADD CONSTRAINT "orgs_auto_top_up_threshold_min" CHECK (("auto_top_up_threshold" >= (10)::numeric AND "auto_top_up_threshold" = trunc("auto_top_up_threshold") AND "auto_top_up_threshold" < 'Infinity'::numeric AND "auto_top_up_threshold" > '-Infinity'::numeric));
```

SQLFluff (4.3.0)

[error] Line is too long (263 > 80).

(LT05)